guide/getting-started/open-design.md

```diff
@@ -50,67 +50,66 @@ Illustration sources
+We could even argue that bitcoin needs open design to stay true to its mission, principles, and philosophy. Most users don’t interact directly with technical protocols but with the applications built on top of it. If those applications are centrally controlled or manipulated, it won’t matter if the protocol is open. So it is up to the community to ensure that bitcoin remains [decentralized](/guide/foundations/principles/#decentralization) and allows for permissionless participation at the open-source application layer.
 
 ## Values
 
 In open design, we:
 
-- collaborate and don't compete
-- are transparent and share our process and work
-- make it easy for others to use and build on our ideas and work
-- are inclusive and help others
-- are open-minded and flexible
-- work multidisciplinary because good results require different perspectives and skillsets
-- continuously learn independently and from each other
-- rely on our peer group for free and communal validation of quality
-- are accountable for our actions and have vested interest in the groups' actions and goals
+- collaborate without competing
+- share our process and work in a transparent way
+- make it easy for others to build on our work and ideas
+- aim for inclusivity
+- approach problems with an open-mind
+- embrace multidisciplinary feedback
+- rely on our peers for quality assurance and validation
+- fulfill the open design group’s goals through action
+
+
+We value:
 
-And we value:
+- moral and responsible choices
+- products that solve real problems and address real needs
+- usability accessibility for the broadest possible range of people, locations, and backgrounds
+- highly functional aesthetic elements that influence both how people feel and think
 
-- design ethics to make moral and responsible choices
-- human design to create products for real needs and desires
-- usability to create products that allow users to solve their problems effectively
-- accessibility to make products usable by people with the broadest possible range of abilities, in the widest possible range of situations
-- aesthetics, which has a highly functional element and also influences how people feel and think
 
-## Getting started
+## Getting
+
+If you are new to open design for bitcoin, which many are, it can be tricky to find a place to start. Not only is the tech complex, but there are few educational resources available. The  [Contribute page]({{ '/contribute' | relative_url }}) is a good starting point. These tips can also help you find your footing:
```

Comment:
```suggestion
If you are new to open design for Bitcoin, which many are, it can be tricky to find a place to start. Not only is the tech complex, but there are few educational resources available. The [contribute page]({{ '/contribute' | relative_url }}) is a good starting point. These tips can also help you find your footing:
```